sql/core/src/test/resources/sql-tests/inputs/pgSQL/date.sql

```diff
@@ -228,9 +228,9 @@ SELECT f1 - date '2000-01-01' AS `Days From 2K` FROM DATE_TBL;
 -- test extract!
 --
 -- epoch
---
--- SELECT EXTRACT(EPOCH FROM DATE        '1970-01-01');     --  0
--- SELECT EXTRACT(EPOCH FROM TIMESTAMP   '1970-01-01');     --  0
+
```

Comment:
Let's remove this new empty line. It seems that you replaced `--` with this.

Reply:
reverted to --